File: Data Structures & Algorithms/concatenation-of-array/submission-9.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int len = nums.length;
        int[] retVal = new int[len*2];

        int left = len;

        for (int i = 0; i < len; i++, left++){
            retVal[i] = nums[i];
            retVal[left] = nums[i];
        }
        return retVal;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-9.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        seen = []
        longest = 0

        for num in nums:
            l, r = num - 1, num + 1
            counter = 1
            while num not in seen and l in nums or r in nums:
                if l in nums:
                    seen.append(l)
                    l -= 1
                    counter += 1
                elif r in nums:
                    seen.append(r)
                    r += 1
                    counter += 1 
            longest = max(counter, longest)
        
        return longest
                    

